plugins/inputs/logparser/logparser_test.go

```diff
@@ -148,23 +151,21 @@ func TestGrokParseLogFilesAppearLater(t *testing.T) {
 		},
 		map[string]string{
 			"response_code": "200",
-			"path":          emptydir + "/test_a.log",
+			"path":          filepath.Join(emptydir, "test_a.log"),
 		})
 }
 
 // Test that test_a.log line gets parsed even though we don't have the correct
 // pattern available for test_b.log
 func TestGrokParseLogFilesOneBad(t *testing.T) {
-	thisdir := getCurrentDir()
-
 	logparser := &LogParserPlugin{
 		Log:           testutil.Logger{},
 		FromBeginning: true,
-		Files:         []string{thisdir + "testdata/test_a.log"},
+		Files:         []string{filepath.Join(testdataDir, "test_a.log")},
 		GrokConfig: GrokConfig{
 			MeasurementName:    "logparser_grok",
 			Patterns:           []string{"%{TEST_LOG_A}", "%{TEST_LOG_BAD}"},
-			CustomPatternFiles: []string{thisdir + "testdata/test-patterns"},
+			CustomPatternFiles: []string{filepath.Join(testdataDir, "test-patterns")},
```

Comment:
Here you are changing from _current dir_ to _test data dir_. I guess this is intented!?

Reply:
Exactly.

Reply:
Then resolved. :-)